inference-engine/ie_bridges/python/sample/speech_sample/README.md

```diff
@@ -0,0 +1,207 @@
+                        Optional. Write GNA model to file using path/filename
+                        provided.
+  -iname INPUT_LAYERS, --input_layers INPUT_LAYERS
+                        Optional. Layer names for input blobs. The names are
+                        separated with ",". Allows to change the order of
+                        input layers for -i flag. Example: Input1,Input2
+  -oname OUTPUT_LAYERS, --output_layers OUTPUT_LAYERS
+                        Optional. Layer names for output blobs. The names are
+                        separated with ",". Allows to change the order of
+                        output layers for -o flag. Example:
+                        Output1:port,Output2:port.
+```
+
+## Model Preparation
+
+You can use the following model optimizer command to convert a Kaldi nnet1 or nnet2 neural network to Inference Engine Intermediate Representation format:
+
+```sh
+python mo.py --framework kaldi --input_model wsj_dnn5b.nnet --counts wsj_dnn5b.counts --remove_output_softmax --output_dir <OUTPUT_MODEL_DIR>
+```
+
+The following pre-trained models are available:
+
+- wsj_dnn5b_smbr
+- rm_lstm4f
+- rm_cnn4a_smbr
+
+All of them can be downloaded from [https://storage.openvinotoolkit.org/models_contrib/speech/2021.2](https://storage.openvinotoolkit.org/models_contrib/speech/2021.2).
+
+## Speech Inference
+
+You can do inference on Intel® Processors with the GNA co-processor (or emulation library):
+
+```sh
+python speech_sample.py -d GNA_AUTO -m wsj_dnn5b.xml -i dev93_10.ark -r dev93_scores_10.ark -o result.npz
+```
+
+> **NOTES**:
+>
+> - Before running the sample with a trained model, make sure the model is converted to the Inference Engine format (\*.xml + \*.bin) using the [Model Optimizer tool](../../../../../docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md).
+>
+> - The sample accepts models in ONNX format (.onnx) that do not require preprocessing.
+>
+> - The sample supports input and output in numpy file format (.npz)
+
+## Sample Output
+
+The sample application logs each step in a standard output stream.
+
+```sh
+[ INFO ] Creating Inference Engine
+[ INFO ] Reading the network: wsj_dnn5b.xml
+[ INFO ] Configuring input and output blobs
+[ INFO ] Using scale factor of 2175.4322417974627 calculated from first utterance.
+[ INFO ] Loading the model to the plugin
+[ INFO ] Starting inference in synchronous mode
+[ INFO ] Utterance 0 (4k0c0301)
+[ INFO ] Frames in utterance: 1294
+[ INFO ] Total time in Infer (HW and SW): 5305.47ms
+[ INFO ] max error: 0.7051839828491211
+[ INFO ] avg error: 0.044838781794944625
+[ INFO ] avg rms error: 0.05823873597800535        
+[ INFO ] stdev error: 0.03716495682038603
+[ INFO ]
+[ INFO ] Utterance 1 (4k0c0302)
+[ INFO ] Frames in utterance: 1005
+[ INFO ] Total time in Infer (HW and SW): 5031.53ms
+[ INFO ] max error: 0.7575974464416504
+[ INFO ] avg error: 0.04521660570793721
+[ INFO ] avg rms error: 0.05860129608000006
+[ INFO ] stdev error: 0.03727694288040223
```

Comment:
Please add formatting for all floating points to align with C++ sample

Reply:
do you mean print only 7 digits after the point?

Reply:
@generalova-kate yes

Reply:
ok